File: src/main/java/cn/xihan/http/config/IRequestCache.java
package cn.xihan.http.config;


import cn.xihan.http.model.CacheMode;

/**
 *    author : Android 轮子哥
 *    github : <a href="https://github.com/getActivity/EasyHttp">https://github.com/getActivity/EasyHttp</a>
 *    time   : 2021/05/22
 *    desc   : 请求缓存配置
 */
public interface IRequestCache {

    /**
     * 获取缓存的模式
     */

    CacheMode getCacheMode();

    /**
     * 获取缓存的有效时长（以毫秒为单位）
     */
    long getCacheTime();
}

File: src/main/java/cn/xihan/lib/easyhttp/config/IRequestCache.kt
package cn.xihan.lib.easyhttp.config

import cn.xihan.lib.easyhttp.model.CacheMode

/**
 * author : Android 轮子哥
 * github : [https://github.com/getActivity/EasyHttp](https://github.com/getActivity/EasyHttp)
 * time   : 2021/05/22
 * desc   : 请求缓存配置
 */
interface IRequestCache {
    /**
     * 获取缓存的模式
     */
    val cacheMode: CacheMode?

    /**
     * 获取缓存的有效时长（以毫秒为单位）
     */
    val cacheTime: Long
        get() = Long.MAX_VALUE
}